spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/jms/ConnectionStringResolver.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.cloud.autoconfigure.jms;
+
+import java.util.HashMap;
+
+public class ConnectionStringResolver {
```

Comment:
1. Make `resolve()` as static method, which directly return the map.
2. Extract constant as class level constant.